### PR TITLE
Move integration test artifacts into primary source tree

### PR DESCRIPTION
**Pull request by bmc-msft**

### CODE REVIEW

**Comment by bmc-msft** [comment]
This is a step towards moving `check-pr` off of my plate and into the work of a bot.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -312,3 +312,77 @@ jobs:
...
+          Invoke-Expression (Invoke-RestMethod 'https://chocolatey.org/install.ps1')
+          choco install llvm
+          choco install make
+          $env:Path += ";C:\Program Files\LLVM\bin;C:\ProgramData\chocolatey\bin"
+          
+          
+          cd src/integration-tests
+          
+          mkdir artifacts/windows-libfuzzer
+          cd libfuzzer
```
Comment (ranweiler) [inline .github/workflows/ci.yml:368]: FYI, PS has [`Push-Location`](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/push-location?view=powershell-7.1) and `Pop-Location`, which I believe get `pushd` and `popd` aliases by default.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -312,3 +312,77 @@ jobs:
...
+    steps:
+    - uses: actions/checkout@v2
+    - run: |
+          Set-ExecutionPolicy Bypass -Scope Process -Force 
+          $ProgressPreference = 'SilentlyContinue'
+          Invoke-Expression (Invoke-RestMethod 'https://chocolatey.org/install.ps1')
+          choco install llvm
+          choco install make
+          $env:Path += ";C:\Program Files\LLVM\bin;C:\ProgramData\chocolatey\bin"
+
```
Comment (ranweiler) [inline .github/workflows/ci.yml:363]:
```suggestion
```

**Review by ranweiler — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bmc-msft) [reply .github/workflows/ci.yml:367]: I have more tests to add in the future.  I'll try that out then and rework this.

**Pull request merged**